packer/http/centos-6.6/ks-ganeti.cfg

```diff
@@ -66,7 +66,10 @@ man-pages
 
 %post
 # update root certs
-wget -O/etc/pki/tls/certs/ca-bundle.crt https://hg.mozilla.org/releases/mozilla-release/raw-file/default/security/nss/lib/ckfw/builtins/certdata.txt
+wget https://raw.githubusercontent.com/bagder/curl/master/lib/mk-ca-bundle.pl
+chmod +x mk-ca-bundle.pl
```

Comment:
No need to call `chmod` if you're just going to run the `perl` binary instead of executing the script directly.